lib/modules/manager/sbt/extract.spec.ts

```diff
@@ -378,5 +378,27 @@ describe('modules/manager/sbt/extract', () => {
         packageFileVersion: undefined,
       });
     });
+
+    it('extract sbt version', () => {
+      expect(
+        extract(
+          `sbt.version=1.6.0
```

Comment:
can this file have more than one version definition? then we should set the `replaceString` to the full string, so auto replace doesn't wrongly replace same version of another dependency.

https://github.com/renovatebot/renovate/blob/78a384b9eede13ddb0f5380afd8c597190de1016/lib/workers/repository/update/branch/auto-replace.ts#L175

Reply:
This file can only have a single definition.
Should I still define the `replaceString`?

Reply:
yes please, maybe in future there are more version configs inside that file.

Reply:
I've had to change a bit the regex to be multiline then and extract the matching line.

I'm not experienced in TS, if there's a better way do not hesitate to tell me.